diagnostic.md

```diff
@@ -157,7 +176,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+Assuming the Divs fit on one line the order from left to right would be:
+divTwo divThree divFour divFive divOne
 <!-- your answer ends here -->
```

Comment:
Close, .divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).